src/commands/readme.ts

```diff
@@ -221,7 +221,7 @@ USAGE
   private commandPath(plugin: Interfaces.Plugin, c: Command.Cached): string | undefined {
     const commandsDir = plugin.pjson.oclif.commands
     if (!commandsDir) return
-    let p = path.join(plugin.root, commandsDir, ...c.id.split(':'))
+    let p = path.join(plugin.root, commandsDir, ...c.id.split(plugin.pjson.oclif.topicSeparator ?? ':'))
```

Comment:
`plugin.pjson.oclif.topicSeparator` instead of `this.config.topicSeparator` because we want the sep in the CLI, not the one from `oclif`. Default to `:` if not defined.

Reply:
The `id` property should _always_ be separated by colons, meaning you shouldn't have to lookup the topicSeparator here 

It looks like the actual issue here is that the [`formatCommand`](https://github.com/oclif/core/blob/61aa44e5ae5e200d5df69433fdf1b8bd6cd9f2dc/src/help/index.ts#L242) method in oclif/core is mutating the command id to respect the configured `topicSeparator`.

You could consider updating `formatCommand` to make a copy of the command before altering the `id`, although I don't know if that would have downstream consequences in `Help`. A simpler solution might be to make a copy of the command before passing it to `formatCommand` on line 179

Reply:
Ahh, good catch. I made it copy the command here instead of in oclif/core 👍🏼.